app/scripts/views/complete_sign_up.js

```diff
@@ -212,31 +212,57 @@ define(function (require, exports, module) {
     },
 
     /**
-     * Check if the user is eligible to send an _isEligibleToSendSms
+     * Check if the user is eligible to send an sms
      *
      * @param {Object} verifiedAccount - account that was just verified.
-     * @returns {Promise} - resolves to `true` if user can send an SMS,
-     *  `false` otw.
+     * @returns {Promise} resolves to an object with:
+     *   * {Boolean} ok - true if user can send an SMS
+     *   * {String} country - user's country
      * @private
      */
-    _isEligibleToSendSms (verifiedAccount) {
+    _smsStatus (verifiedAccount) {
       return p().then(() => {
-        return ! this.isSignIn() &&
-               // If already on a mobile device, doesn't make sense to send an SMS.
-               ! this.getUserAgent().isAndroid() &&
-               ! this.getUserAgent().isIos() &&
-               this.isInExperimentGroup('sendSms', 'treatment') &&
-               // If a user is already signed in to Sync which is different to the
-               // user that just verified, show them the old "Account verified!" screen.
-               ! this._isAnotherUserSignedIn(verifiedAccount) &&
-               // The auth server can gate whether users can send an SMS based
-               // on the user's country and whether the SMS provider account
-               // has sufficient funds.
-               verifiedAccount.smsStatus();
+        if (this.isSignIn() ||
+          ! this.isInExperimentGroup('sendSms', 'treatment') ||
+            // If already on a mobile device, doesn't make sense to send an SMS.
+            this.getUserAgent().isAndroid() ||
+            this.getUserAgent().isIos() ||
+            // If a user is already signed in to Sync which is different to the
+            // user that just verified, show them the old "Account verified!" screen.
+            this._isAnotherUserSignedIn(verifiedAccount)) {
+          return p({ ok: false });
+        }
+
+        // The auth server can gate whether users can send an SMS based
+        // on the user's country and whether the SMS provider account
+        // has sufficient funds.
+        return verifiedAccount.smsStatus(this.relier.pick('country'))
+          .then((resp) => {
+            if (resp.country === 'RO' && resp.ok) {
```

Comment:
Since this is the logic most likely to change, we might be able to move this into fxa-content-experiments.